search/api.py

```diff
@@ -43,15 +47,20 @@ def perform_search(
         size=10,
         from_=0,
         course_id=None):
-    """ Call the search engine with the appropriate parameters """
+    """
+    Call the search engine with the appropriate parameters
+    """
     # field_, filter_ and exclude_dictionary(s) can be overridden by calling application
     # field_dictionary includes course if course_id provided
-    (field_dictionary, filter_dictionary, exclude_dictionary) = SearchFilterGenerator.generate_field_filters(
-        user=user,
-        course_id=course_id
```

Comment:
Why are these lines changing?
The preferred method from the style guide for breaking long lines is to place each `arg`/`kwarg` on its own line; why undo it?
This _seems_ like unneeded code churn?

Reply:
Done. I returned it as like it was